sims.code-workspace

```diff
@@ -0,0 +1,93 @@
+{
+  "folders": [
```

Comment:
As a suggestion we could have an entry for `sources` because that is the folder where we should be to use the Makefile.